libs/wingc/turbo.json

```diff
@@ -11,7 +11,7 @@
       ]
     },
     "test": {
-      "dependsOn": ["compile", "@winglang/sdk#compile", "examples-valid#topo", "examples-invalid#topo"]
+      "dependsOn": ["compile", "@winglang/sdk#compile", "examples-valid#topo", "examples-invalid#topo", "jsii-fixture#compile"]
```

Comment:
Are you sure this is needed? `examples-valid#topo` is already dependent on `jsii-fixture#compile`

Reply:
Ah good catch - I think I might have just ran the wrong commannd...